README.md

```diff
@@ -22,6 +22,13 @@ Previous versions of ODH relied on [JupyterHub](jupyterhub/README.md) for managi
 
 Any components that were removed with the update to ODH 1.4 have been relocated to the [ODH Contrib](https://github.com/opendatahub-io-contrib) organization under the [odh-contrib-manifests](https://github.com/opendatahub-io-contrib/odh-contrib-manifests) repo.  You can reference the [odh-contrib kfdef](kfdef/odh-contrib.yaml) as a reference on how to deploy any of the odh-contrib-manifests components
 
+### Component Versions
+
+| Manifest Version | Operator Version | ODH Dashboard | ODH Notebook Controller | ODH Notebook Images | Data Science Pipelines | ModelMesh |
+| ---------------- | ---------------- | ------------- | ----------------------- | ------------------- |----------------------- | --------- |
+| v1.4.1 | v1.4.1 | v2.5.2 | ??? | v1.3.1 | v1.2.1 | v0.9.0 |
```

Comment:
@VaishnaviHire @rimolive @andrewballantyne fyi.

Reply:
Sweet! It's good to see what version everything is at. Is the Manifest Version really different than the Operator Version?

Reply:
Notebook-controller version is v1.6 since the current notebook-controller image is built off of the [v1.6 release branch](https://github.com/opendatahub-io/kubeflow/tree/v1.6-branch/components/odh-notebook-controller)

@strangiato I don't think we need to include the versions for odh-manifests and odh-operator since 
1. They will be the same number
2. We should be keeping the component versions up to date so anyone pulling the manifests from the default branch know what versions are included.

I think we can push document the compatibility matrix for ODH Version --> Component version in the GitHub  and opendatahub-io release notes.

Reply:
I made some updates based on the feedback.  

I dropped the Operator Version.  I originally included it because I know there were some manifest releases in the past that did not include a separate operator version.  Sounds like that won't be the case moving forward thought.

Updated the notebook controller version for both 1.4.1 and 1.4.0.  It didn't appear that anything changed between the the so I assume 1.4.0 was also running 1.6.

I also updated the notebook images for 1.4.0 to be N/A.  It looks like with 1.4.1 the goal was to introduce the pre-built images, so I assume it probably doesn't make sense to include a version number for this in 1.4.0.

Let me know if there are any other changes you think would be relevant.

Reply:
@strangiato I don't know if we need `Manifest Version` in the table since the compatibility matrix would be incorrect between releases of odh-manifests.

Reply:
@LaVLaS Would you prefer to update that column to `Operator Version` instead?  I'm not quite sure what you mean by "the compatibility matrix would be incorrect between releases of odh-manifests."

Would you want to add a row for`master` as well to document what the master branch is currently at?

Reply:
@strangiato Apologies. I just realized that the table is trying to list the component versions for every released version of ODH (1.4.0, 1.4.1, ..., 1.4.z, 1.5.z, ...).  I was thinking it just showed the current component versions in the `master` branch.  

I'll defer to community opinion if we need a table to show older versions in the README if they are available as part of the release notes

Reply:
I see no problem -- but maybe we mark the current latest as "master" version and just copy it down a line in the table and rename that "master" version to whatever we release for the Operator.